go.mod

```diff
@@ -30,6 +30,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf // indirect
```

Comment:
Can you please move this inside the existing group of indirect dependencies instead of putting it on an individual line.